<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowMain</class>
 <widget class="QMainWindow" name="WindowMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dubbing Tools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>781</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Project</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lneProjectPath">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbSelectProject">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Languages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leLanguagesList">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpenProject">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenProject"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCopyLanguage"/>
    <addaction name="actionDeleteLanguage"/>
    <addaction name="actionExportSubtitles"/>
   </widget>
   <addaction name="menuOpenProject"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCopyLanguage">
   <property name="text">
    <string>Copy Language</string>
   </property>
  </action>
  <action name="actionExportSubtitles">
   <property name="text">
    <string>Export subtitles</string>
   </property>
  </action>
  <action name="actionDeleteLanguage">
   <property name="text">
    <string>Delete Language</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
